packages/components/src/components/form/Select/index.tsx

```diff
@@ -132,9 +144,18 @@ const Select = ({
     width,
     variant = 'large',
     noTopLabel = false,
+    useKeyPressScroll,
+    isSearchable = false,
     ...props
 }: Props) => {
+    // TODO find proper type
+    const selectRef: React.Ref<any> = React.useRef(null);
```

Comment:
Proper type here would be welcome. Also you can omit `React.` and simply use `useRef` as it is imported above.